### PR TITLE
docs/adr: Update March 2022 testnet ADR to clarify bridge node

### PR DESCRIPTION
**Pull request by renaynay**
Related to confusion in #548

### CODE REVIEW

File: docs/adr/adr-003-march2022-testnet.md [hunk 1]
```diff
@@ -20,11 +21,17 @@ Refers to the data availability "halo" network created around the Core network.
...
-instead of reconstructing blocks via downloading enough shares from the network, it receives headers and blocks directly from its 
-trusted Core node, validating blocks, erasure coding them, and producing `ExtendedHeader`s to broadcast to the Celestia 
-DA network.
+A **bridge** node is a node that is connected to a celestia-core node via RPC. It receives a remote address from a 
+running celestia-core node and listens for blocks from celestia-core. For each new block from celestia-core, the **bridge**
+node performs basic validation on the block via `ValidateBasic()`, extends the block data, generates a Data Availability 
+Header (DAH) from the extended block data, and creates an `ExtendedHeader` from the block header and the DAH, and finally 
+broadcasts it to the data availability network (DA network).
+
+A **bridge** node does not care about what kind of celestia-core node it is connected to (validator or regular full node),
```
[the remark this answers is not in the document]
Comment (liamsi) [reply docs/adr/adr-003-march2022-testnet.md:30]: > ? I mean what is the difference between validator and "regular full node"?

see first and last node type here: https://docs.tendermint.com/master/nodes/#node-types

It exists and the main diff is that one actively signs and the other does not.

[the remark this answers is not in the document]
Comment (Wondertan) [reply docs/adr/adr-003-march2022-testnet.md:30]: Ok, so the confusion I had here is that full nodes do full validation and semantically validators, while validators do the same + block production, and their name doesn't semantically include this trait.

It's orthogonal to the PR, just clarifications. Resolve freely

**Review by adlerjohn — APPROVED** [review]

**Review by YazzyYaz — APPROVED** [review]
LGTM, once it's merged, I can update the definition in the docs for Bridge Nodes.

**Review by liamsi — APPROVED** [review]
LGTM. I'm wondering if we should again tag @musalbas on this in the context of https://github.com/celestiaorg/celestia-node/issues/548

**Pull request merged**